include/dlaf_c/init.h

```diff
@@ -0,0 +1,27 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <dlaf_c/utils.h>
+
+/// Initialize pika runtime and DLA-Future
+///
+/// The pika runtime is automatically suspended within this function
```

Comment:
Make post condition (with `@post`)?

Reply:
Started going through the documentation and added special keywords where sensible (including this one).